.drone.yml

```diff
@@ -191,6 +191,104 @@ volumes:
   host:
     path: /var/run/docker.sock
 
+---
+kind: pipeline
+name: default-s390x
+
+platform:
+  os: linux
+  arch: amd64
+
+node:
+  arch: s390x
+
+steps:
+- name: fetch
+  image: ubuntu:18.04
+  commands:
+  - apt-get update
+  - apt-get install -y git
+  - git fetch --tags
+
+- name: build
+  pull: default
+  image: rancher/dapper:v0.5.8
+  commands:
+  - dapper ci
+  privileged: true
+  volumes:
+  - name: socket
+    path: /var/run/docker.sock
+
+- name: publish-image
+  image: plugins/docker
```

Comment:
This should be `rancher/drone-images:docker-s390x`.

Reply:
I created https://github.com/longhorn/backing-image-manager/pull/51 to change the image.
Sorry for the mistake.